mapping/mapping_vectors.go

```diff
@@ -202,6 +208,12 @@ func validateVectorFieldAlias(field *FieldMapping, parentName string,
 			reflect.ValueOf(index.SupportedSimilarityMetrics).MapKeys())
 	}
 
+	if _, ok := index.SupportedVectorIndexOptimizations[field.VectorIndexOptimizedFor]; !ok {
+		return fmt.Errorf("field: '%s', invalid optimization "+
```

Comment:
Let's not error out, with future in mind if we add more options.

Reply:
I thought updating index.SupportedVectorIndexOptimizations in bleve_index_api and its version in bleve's go.mod would take care of this?

Reply:
What do you mean!? I'm talking about the case in the future - when we add more optimizations and there's a mixed cluster scenario where some nodes won't support it.

Reply:
Ah right.
Removed this snippet so there's no check at this point for valid optimisation metrics.